src/Spline.cpp

```diff
@@ -200,10 +200,25 @@ double base::geometry::SplineBase::getCurveLength(double relative_resolution) co
     return length;
 }
 
-double SplineBase::getUnitParameter()
```

Comment:
If the new code is as broken as the existing code, then leave the existing code. At least, it does not change the class behaviour.